### PR TITLE
docs(src/utilities): Use `describe()` instead of `@alt`

### PR DESCRIPTION
**Pull request by Zearin**
Addresses https://github.com/processing/p5.js/issues/5139. Attention @lm-n. ;-)

Changes:
Use `describe()` over `@alt` in inline docs within 📁 src/utilities/.

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md

### CODE REVIEW

**Comment by Zearin** [comment]
@lm-n ping 😇

File: src/utilities/conversion.js [hunk 1]
```diff
@@ -24,15 +24,13 @@ import p5 from '../core/main';
  * let str = '20';
  * let diameter = float(str);
  * ellipse(width / 2, height / 2, diameter, diameter);
+ * describe(`20×20 white ellipse in the center of the canvas`);
```
Comment (lm-n) [inline src/utilities/conversion.js:27]: can you change text to 20 by 20. Screen readers would read "20x20" as twenty ex twenty

**Review by lm-n — CHANGES_REQUESTED** [review]
@Zearin great job! could you change those that have numberxnumber to number by number. I added a comment on #5596 after I committed, it'd be great if you could also fix it.

File: src/utilities/conversion.js (hunk 1 above)
Comment (Zearin) [reply src/utilities/conversion.js]: Fixed in 9c0e4fb7

**Comment by Qianqianye** [comment]
Thanks @Zearin and @lm-n!

**Pull request merged**